NORSE_actor/externalNORSERun.py

```diff
@@ -13,17 +13,23 @@
 import dimensionCheck
 import pGridMode
 import matlabDouble
-import EHat
+import EHat_calc
 import CoulombLogarithm
+import hdf5Write
+
 import numpy as np
 import matlab.engine
-
 import itertools
 import pickle
 import ual
 from time import asctime
 import copy
 
+# Define function to get variable name as string
+def get_name(var):
+	import itertools
+	return [tpl[0] for tpl in 
+	itertools.ifilter(lambda x: var is x[1], globals().items())][0]
 

```

Comment:
This might be dangerous to identify an array based on its values (and size). For numpy arrays this might be a better solution: https://stackoverflow.com/questions/34980833/python-name-of-np-array-variable-as-string Should test and implement if ok.

Reply:
Done.